vs2010.vcxproj.static.template

```diff
@@ -0,0 +1,223 @@
+      <Optimization>Disabled</Optimization>
+      <FavorSizeOrSpeed>Speed</FavorSizeOrSpeed>
+      <PreprocessorDefinitions>APR_DECLARE_STATIC;WIN32_LEAN_AND_MEAN;WIN64;_DEBUG;_WINDOWS;HAVE_OPENSSL;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <MinimalRebuild>true</MinimalRebuild>
+      <ExceptionHandling>
+      </ExceptionHandling>
+      <BasicRuntimeChecks>EnableFastChecks</BasicRuntimeChecks>
+      <RuntimeLibrary>MultiThreadedDebugDLL</RuntimeLibrary>
+      <RuntimeTypeInfo>false</RuntimeTypeInfo>
+      <PrecompiledHeader>
+      </PrecompiledHeader>
+      <WarningLevel>Level3</WarningLevel>
+      <DebugInformationFormat>EditAndContinue</DebugInformationFormat>
+      <CallingConvention>Cdecl</CallingConvention>
+    </ClCompile>
+    <Link>
+      <AdditionalDependencies>mswsock.lib;Rpcrt4.lib;Psapi.lib;Shlwapi.lib;Ws2_32.lib;apr-1.lib;@SSL_LIBS@;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalLibraryDirectories>@SSL_LIB_DIR@;@APR_LIB_DIR@;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <GenerateDebugInformation>true</GenerateDebugInformation>
+      <SubSystem>Windows</SubSystem>
+      <TargetMachine>MachineX64</TargetMachine>
+    </Link>
+  </ItemDefinitionGroup>
+  <ItemGroup>
+    <ClCompile Include=".\src\address.c" />
+    <ClCompile Include=".\src\bb.c" />
+    <ClCompile Include=".\src\dir.c" />
+    <ClCompile Include=".\src\error.c" />
+    <ClCompile Include=".\src\file.c" />
+    <ClCompile Include=".\src\info.c" />
+    <ClCompile Include=".\src\jnilib.c" />
+    <ClCompile Include=".\src\lock.c" />
+    <ClCompile Include=".\src\misc.c" />
+    <ClCompile Include=".\src\mmap.c" />
+    <ClCompile Include=".\src\multicast.c" />
+    <ClCompile Include=".\src\network.c" />
+    <ClCompile Include=".\src\os.c" />
+    <ClCompile Include=".\src\os_unix_system.c" />
+    <ClCompile Include=".\src\os_unix_uxpipe.c" />
+    <ClCompile Include=".\src\os_win32_ntpipe.c" />
+    <ClCompile Include=".\src\os_win32_registry.c" />
+    <ClCompile Include=".\src\os_win32_system.c" />
+    <ClCompile Include=".\src\poll.c" />
+    <ClCompile Include=".\src\pool.c" />
+    <ClCompile Include=".\src\proc.c" />
+    <ClCompile Include=".\src\shm.c" />
+    <ClCompile Include=".\src\ssl.c" />
+    <ClCompile Include=".\src\sslcontext.c" />
+    <ClCompile Include=".\src\sslinfo.c" />
+    <ClCompile Include=".\src\sslnetwork.c" />
+    <ClCompile Include=".\src\sslutils.c" />
+    <ClCompile Include=".\src\stdlib.c" />
+    <ClCompile Include=".\src\thread.c" />
+    <ClCompile Include=".\src\user.c" />
+  </ItemGroup>
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
+  <ImportGroup Label="ExtensionTargets">
+  </ImportGroup>
+</Project>
```

Comment:
Actually, I'd like to keep this as-is.  It matches the "real" vcxproj file under `openssl-dynamic`.  It's how Visual Studio saves the project file.